tests/browser.js

```diff
@@ -0,0 +1,195 @@
+/* globals describe, before, it */
+const importer = require('./../src')
+const IPFSRepo = require('ipfs-repo')
+const BlockService = require('ipfs-blocks').BlockService
+const DAGService = require('ipfs-merkle-dag').DAGService
+const DAGNode = require('ipfs-merkle-dag').DAGNode
+const UnixFS = require('ipfs-unixfs')
+
+const async = require('async')
+const store = require('idb-plus-blob-store')
+const _ = require('lodash')
+const repoContext = require.context('buffer!./repo-example', true)
+
+const FixedSizeChunker = require('./../src/chunker-fixed-size')
+const expect = require('chai').expect
+const stringToStream = require('string-to-stream')
+const through = require('through2')
+
+const myFile = require('buffer!./test-data/1MiB.txt')
+const fileStream = function () {
+  return stringToStream(myFile)
+}
+global.Buffer = global.Buffer || require('buffer').Buffer
+
+const smallBuf = require('buffer!./test-data/200Bytes.txt')
+const bigBuf = require('buffer!./test-data/1.2MiB.txt')
+const bigBlock = require('buffer!./test-data/1.2MiB.txt.block')
+const bigLink = require('buffer!./test-data/1.2MiB.txt.link-block0')
+const marbuf = require('buffer!./test-data/200Bytes.txt.block')
+
+const idb = window.indexedDB ||
+  window.mozIndexedDB ||
+  window.webkitIndexedDB ||
+  window.msIndexedDB
+
+idb.deleteDatabase('ipfs')
+idb.deleteDatabase('ipfs/blocks')
+
+describe('IPFS data importing tests on the Browser', function () {
+  before(function (done) {
+    this.timeout(10000)
+    var repoData = []
+    repoContext.keys().forEach(function (key) {
+      repoData.push({
+        key: key.replace('./', ''),
+        value: repoContext(key)
+      })
+    })
+
+    const mainBlob = store('ipfs')
+    const blocksBlob = store('ipfs/blocks')
+
+    async.eachSeries(repoData, (file, cb) => {
```

Comment:
can you make a file to be the set up of things for the browser and then another for the tests? a la js-ipfs and friends :)